examples/examples.cmake

```diff
@@ -5,6 +5,8 @@
 
 add_executable(decode_oneshot ${CMAKE_CURRENT_LIST_DIR}/decode_oneshot.cc)
 target_link_libraries(decode_oneshot jxl_dec jxl_threads)
+add_executable(decode_progressive ${CMAKE_CURRENT_LIST_DIR}/decode_progressive.cc)
```

Comment:
Add this to the CMakeLists.txt too; CMakeLists.txt shows how to write an example project that uses libjxl.